Api/db/seeds.rb

```diff
@@ -19,522 +18,150 @@
-        shift_id: 1 # mañana
-      },
-      {
-        date: "14-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "15-07-19",
-        shift_id: 4 # off
-      },
-      {
-        date: "16-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "17-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "18-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "19-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "20-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "21-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "22-07-19",
-        shift_id: 4 # off
-      },
-      {
-        date: "23-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "24-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "25-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "26-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "27-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "28-07-19",
-        shift_id: 1 # mañana
-      },
-      {
-        date: "29-07-19",
-        shift_id: 4 # off
-      },
-      {
-        date: "30-07-19",
-        shift_id:  1 # mañana
-      },
-      {
-        date: "31-07-19",
-        shift_id:  1 # mañana
-      }
+             { date: "2019/07/01", shift_id: 4 },
+             { date: "2019/07/02", shift_id: 1 },
+             { date: "2019/07/03", shift_id: 1 },
+             { date: "2019/07/04", shift_id: 1 },
+             { date: "2019/07/05", shift_id: 1 },
+             { date: "2019/07/06", shift_id: 1 },
+             { date: "2019/07/07", shift_id: 1 },
+             { date: "2019/07/08", shift_id: 4 },
+             { date: "2019/07/09", shift_id: 1 },
+             { date: "2019/07/10", shift_id: 1 },
+             { date: "2019/07/11", shift_id: 1 },
+             { date: "2019/07/12", shift_id: 1 },
+             { date: "2019/07/13", shift_id: 1 },
+             { date: "2019/07/14", shift_id: 1 },
+             { date: "2019/07/15", shift_id: 4 },
+             { date: "2019/07/16", shift_id: 1 },
+             { date: "2019/07/17", shift_id: 1 },
+             { date: "2019/07/18", shift_id: 1 },
+             { date: "2019/07/19", shift_id: 1 },
+             { date: "2019/07/20", shift_id: 1 },
+             { date: "2019/07/21", shift_id: 1 },
+             { date: "2019/07/22", shift_id: 4 },
+             { date: "2019/07/23", shift_id: 1 },
+             { date: "2019/07/24", shift_id: 1 },
+             { date: "2019/07/25", shift_id: 1 },
+             { date: "2019/07/26", shift_id: 1 },
+             { date: "2019/07/27", shift_id: 1 },
+             { date: "2019/07/28", shift_id: 1 },
+             { date: "2019/07/29", shift_id: 4 },
+             { date: "2019/07/30", shift_id: 1 },
+             { date: "2019/07/31", shift_id: 1 }
```

Comment:
I think we should create the other dates to complete the last week of scheduled work  August 01st to August 04th

Reply:
It is a good observation, but the data of August is been considered in schedule_3 and schedule_4.